Realm/RLMResults.h

```diff
@@ -408,6 +427,167 @@ __attribute__((warn_unused_result));
                                                          NSError *_Nullable error))block
                                       keyPaths:(nullable NSArray<NSString *> *)keyPaths
 __attribute__((warn_unused_result));
+
+#pragma mark - Flexible Sync
+
+/**
+ Creates an RLMSyncSubscription matching the RLMResults' local filter.
+
+ This method opens a write transaction that creates or updates a subscription.
+ It is advised to *not* use this method to batch multiple subscription changes
+ to the server.
+ @see: `[RLMSyncSubscription update:queue:onComplete:]` for batch updates
```

Comment:
This wording is confusing to me. This function doesn't do any batching, so it's weird to say that it can do batches and you just shouldn't.